packages/flutter/lib/src/services/platform_channel.dart

```diff
@@ -31,11 +32,12 @@ import 'message_codecs.dart';
 class BasicMessageChannel<T> {
   /// Creates a [BasicMessageChannel] with the specified [name], [codec] and [binaryMessenger].
   ///
-  /// None of [name], [codec], or [binaryMessenger] may be null.
-  const BasicMessageChannel(this.name, this.codec, { this.binaryMessenger = defaultBinaryMessenger })
-    : assert(name != null),
-      assert(codec != null),
-      assert(binaryMessenger != null);
+  /// [name], [codec] cannot be null. The default [ServicesBinding.instance.defaultBinaryMessenger]
```

Comment:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-correct-grammar

So this would be:

```dart
/// The [name] and [codec] arguments cannot be null.
```

Reply:
Updated here and in other places as well.